docroot/themes/custom/uids_base/scss/components/grid.scss

```diff
@@ -0,0 +1,51 @@
+@import "uids/assets/scss/_variables.scss";
+@import "uids/assets/scss/_utilities.scss";
+
+// Determines if the placement is grid worthy based on parent container percentage.
+@mixin inner-grid($parent) {
+  .inner--grid.grid--twocol--50-50 {
+    @if $parent >= 50 {
+      @media screen and (min-width: 768px) {
+        .aggregator-wrapper,
+        .view-content,
+        .list-wrapper {
+          display: grid;
+          column-gap: 3rem;
+          row-gap: $gutter;
+          grid-template-columns: repeat(auto-fill, minmax(46.7%, 1fr));
+          // @todo Remove when uiowa/uiowa#2670 and uiowa/uiowa#2554 are resolved.
+          .card.card--media-left,
+          .card.card--media-right {
+            margin-bottom: 0;
+          }
+        }
+      }
+      @media screen and (min-width: 58.125em) {
+        .aggregator-wrapper,
+        .view-content,
+        .list-wrapper {
+          gap: 3rem;
+          grid-template-columns: 1fr 1fr;
+        }
+      }
+    }
+  }
+  .inner--grid.grid--threecol--33-34-33 {
+    @if $parent >= 75 {
+      @media screen and (min-width: 58.125em) {
```

Comment:
@joewhitsitt Do you think it would help to make some new breakpoint variables for reuse?  Or are these pretty specific to the mixin?

Reply:
i grab this number from the three-col layout sass so that the grid would break down at the same time the three column layout does.